eq-author-api/schema/resolvers/base.js

```diff
@@ -153,6 +154,24 @@ const createQuestionnaire = input => ({
   ...input,
 });
 
+const remapAllNestedIds = entity => {
```

Comment:
Can we put this in utils with a test around it?

Reply:
We need to relocate loads of methods which are located in `base.js`.  I think it's best to save this until we restructure and figure out the final folder structure of the API.